src/store/account/__tests__/actions.test.js

```diff
@@ -0,0 +1,101 @@
+import {
+  logout,
+  getOrderList,
+  addAccountAddress,
+  resetAddressStatus,
+  getCurrentCustomer,
+  getOrderedProductInfo,
+} from '../actions';
+
+describe('account actions', () => {
+  test('getCurrentCustomer()', () => {
+    // Exercise
+    const result = getCurrentCustomer();
+
+    // Verify
+    expect(result).toMatchSnapshot();
```

Comment:
Hi Sanjeev, just a tip for your actions tests, you don't need use a Snapshot in this tests, because the actions just return a object, so you can use some things like:

```
test('getCurrentCustomer()', () => {
    const result = getCurrentCustomer();

    expect(result.type).toEqual(MAGENTO.CURRENT_USER_REQUEST)
})
```

using this way, your tests become more performatics. You can use this in your `actions` and `reducers`.

Reply:
If you wanna a examplo, see this file [actions.test.ts](https://github.com/salomaoluiz/BuyIt/blob/dfa661c1518e5535182f7c46bcd400d50954240f/src/store/notification/__tests__/actions.test.ts)

Reply:
Thank you salomaoliz